### PR TITLE
Little improvements in /members

### PR DESCRIPTION
**Pull request by mllocs**
- Do not hide actions in the manage users page
- Sort members by last_sign_in_at
- Truncate member description
- Hide telephone icon when there is no phone number

### CODE REVIEW

**Comment by mllocs** [comment]
[Texte llistat membres sobresurt](https://trello.com/c/JbYbo4KR/158-texte-llistat-membres-sobresurt)

**Comment by mllocs** [comment]
[Llistat de members endreçat per usuaris més actius primer](https://trello.com/c/2DXlVQnX/154-llistat-de-members-endre%C3%A7at-per-usuaris-m%C3%A9s-actius-primer)

**Comment by mllocs** [comment]
[Al llistat de membre si no hi ha telf o mail que no surti la icona](https://trello.com/c/clXgpH22/157-al-llistat-de-membre-si-no-hi-ha-telf-o-mail-que-no-surti-la-icona)

**Comment by mllocs** [comment]
[members/manage visibilitat a mobile](https://trello.com/c/AbbZJqti/166-members-manage-visibilitat-a-mobile)

File: app/views/users/_member_card.html.erb [hunk 1]
```diff
@@ -15,13 +15,15 @@
...
     <div class="to-member-card__body__description">
-      <%= member.description %>
+      <%= member.description&.truncate(124) %>
     </div>
     <div class="to-member-card__body__items">
-      <div class="to-member-card__body__item">
-        <span class="glyphicon glyphicon-earphone"></span>
-        <%= phone_to member.phone %>
-      </div>
+      <% if member.phone && !member.phone.blank? %>
```
Comment (markets) [inline app/views/users/_member_card.html.erb]: I think `if member.phone.present?` should be enough, wdyt?

**Comment by enricostano** [comment]
Deployed to https://staging.timeoverflow.org

cc/ @sseerrggii

**Comment by sseerrggii** [comment]
Hi, some things to improve: 

* the order of members don't work, maybe is because user_last_sign_in_at belongs to user? 
* Also we want to show the activity date insted of Active Ago like "Active 2 weeks Ago" isn't it
fireshot capture 72 - banco de tiempo local - https___staging timeoverflow org_members
* Also we have some email field empty (internally whatever@example.com) in this case we don't want to show email icon:
fireshot capture 73 - banco de tiempo local - https___staging timeoverflow org_members

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -40,12 +40,16 @@
   describe "GET #index" do
     before { login(user) }
 
-    it 'sorts the users by their member_uid asc by default' do
+    it 'sorts the users by their user_last_sign_in_at desc by default' do
       member.increment!(:member_uid, Member.maximum(:member_uid) + 1)
```
Comment (markets) [inline spec/controllers/users_controller_spec.rb]: I think we can :scissors: this line, it seems

**Comment by markets** [comment]
Closes #426
Closes #425
Closes #439

**Comment by markets** [comment]
Hi @mllocs :wave: we merged #416, where I made some changes to these partials:

- `_user_rows.html.erb` => `_user_row.html.erb`
- `_member_card.html.erb` just wrapped code into `<div class="col-lg-6 col-sm-12">`

Hope it helps with merge conflicts :smiley_cat: :pray:

**Comment by mllocs** [comment]
@markets thanks for the tips!
@sseerrggii rebased and fixed the issues you mentioned :)

**Comment by sseerrggii** [comment]
:green_apple:

**Pull request merged**